lib/search.js

```diff
@@ -5,14 +5,32 @@ var _ = require('lodash');
 module.exports = function(appConfig, date_field) {
     var appConfig = appConfig;
     var date_field = date_field;
-    var logger = appConfig.context.foundation.makeLogger('search_module', 'search_module', {module: 'teraserver_search'});
+    var logger = appConfig.context.foundation.makeLogger({module: 'teraserver_search'});
 
     /*
      TODO: This search functionality is fairly generic but assumes some field names
      date
      type
      */
 
+    function sendError(res, statusCode, query, debug, error, message) {
+        var errResponse = {};
+
+        if (message) {
+            errResponse.message = message
+        }
+
+        if (error) {
+            errResponse.error = error
+        }
+
+        if (debug) {
+            errResponse.criteria = query
```

Comment:
This is just parroting back what the user submitted which isn't what the ticket specified. This was only concerned with ES search errors NOT all errors. The goal here is to show the query as it's being sent to ES and the ES error response.